pkg/cli/cmds/secrets_encrypt.go

```diff
@@ -38,7 +38,11 @@ func NewSecretsEncryptSubcommands(status, enable, disable, prepare, rotate, reen
 			SkipFlagParsing: false,
 			SkipArgReorder:  true,
 			Action:          status,
-			Flags:           EncryptFlags,
```

Comment:
Could we do this as `--output/-o` json/text/etc for improved compatibility with other tools?

Reply:
I would argue that that is overly complex for right now. I don't anticipate a need for outputs other than text and JSON at this time. While it does match kubectl and other k8s tools style, it doesn't really improve compatibility with them.

Reply:
Maybe just make the flag `--output=json` then instead of --json